### PR TITLE
ensure body.value before running additional processes on field value

### PR DESCRIPTION
**Pull request by jjidt**
we were getting 500s when trying to duplicate documents with an empty image field, this add an additional check for a current body.value url before doing the duplication

### CODE REVIEW

File: api/src/coordinators/field.js [hunk 1]
```diff
@@ -183,10 +183,10 @@ const duplicateField = async function(id, originalParentDocId, newParentDocId, o
     return body
   }, {})
   body.orgId = orgId
-
-  if (body.type === FIELD_TYPES.IMAGE || body.type === FIELD_TYPES.FILE) {
+  console.log(body)
```
Comment (jcontonio) [inline api/src/coordinators/field.js]: 👀

**Review by jcontonio — COMMENTED** [review]
Remove that and 👍🏼

File: api/src/coordinators/field.js (hunk 1 above)
Comment (jjidt) [reply api/src/coordinators/field.js]: oops

Comment (jjidt) [reply api/src/coordinators/field.js]: updated

**Review by jcontonio — APPROVED** [review]

**Pull request merged**